tensorflow/python/keras/utils/vis_utils.py

```diff
@@ -141,6 +192,10 @@ def model_to_dot(model,
 
   # Create graph nodes.
   for i, layer in enumerate(layers):
+    if (layer_range is not None) and\
```

Comment:
Just use "if layer_range" instead of "if layer_range is not None" to do the check.

Reply:
done